webui/react/src/pages/ResourcePool/ResourcepoolDetail.tsx

```diff
@@ -152,6 +153,13 @@ const ResourcepoolDetailInner: React.FC = () => {
     return (
       <>
         <JsonGlossary alignValues="right" json={mainSection} translateLabel={camelCaseToSentence} />
+        {!isEmpty(resourceManagerMetadata) && (
+          <Fragment>
+            <Divider />
+            <div className={css.subTitle}>Resource Manager Metadata</div>
+            <JsonGlossary json={resourceManagerMetadata} translateLabel={camelCaseToSentence} />
+          </Fragment>
```

Comment:
nit: use the shorthand here
```suggestion
          <>
            <Divider />
            <div className={css.subTitle}>Resource Manager Metadata</div>
            <JsonGlossary json={resourceManagerMetadata} translateLabel={camelCaseToSentence} />
          </>
```